<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeSelectMenu</class>
 <widget class="QWidget" name="ShapeSelectMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>132</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="selectLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectRectangle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectEllipse">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectPolyline">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Polyline</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectCircle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
